src/utils/prices.ts

```diff
@@ -42,7 +43,7 @@ export function computeTradePriceBreakdown(
 // computes the minimum amount out and maximum amount in for a trade given a user specified allowed slippage in bips
 export function computeSlippageAdjustedAmounts(
   trade: Trade,
-  allowedSlippage: number
+  allowedSlippage: number = INITIAL_ALLOWED_SLIPPAGE
```

Comment:
do we need a default? seems like we'd be hiding a bug

Reply:
Can remove this - had this in before adding the reset in the reducer